File: 백준/Bronze/1934. 최소공배수/최소공배수.java
import java.io.*;
import java.util.*;

public class Main {
    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        int N = Integer.parseInt(br.readLine().trim());
        for (int i = 0; i < N; i++) {
            String[] line = br.readLine().split(" ");
            int A = Integer.parseInt(line[0]);
            int B = Integer.parseInt(line[1]);
            System.out.println(minMultiple(A, B));
        }
    }

    private static int maxAliquot(int A, int B) {
        while (B > 0) {
            int temp = A;
            A = B;
            B = temp % B;
        }
        return A;
    }

    private static int minMultiple(int A, int B) {
        return A * B / maxAliquot(A, B);
    }
}

File: 백준/Bronze/1934. 최소공배수/최소공배수.py
import sys

def maxAliquot(A, B):
    while B > 0:
        A, B = B, A % B
    return A

def minMultiple(A, B):
    return A * B // maxAliquot(A, B)

T = int(sys.stdin.readline().strip())

multiple = 0
aliquot = 0

for _ in range(T):
    A, B = map(int, sys.stdin.readline().split())
    print(minMultiple(A, B))